web/impact/templates/emails/cancel_office_hour_reservation_email_to_mentor.html

```diff
@@ -0,0 +1,18 @@
+Hi {{mentor.first_name}},
+
+Your upcoming office hour session at {{office_hour_date_time|date:"h:m"}} on
+{{office_hour_date_time| date}} with {{startup_name}} was
+cancelled by {{cancelling_party.first_name}} {{cancelling_party.last_name}}. This
```

Comment:
**Non-blocker:** Would it be better to mention `MassChallenge staff canceled the reservation` instead of staff username. In the case, that a staff user cancels the reservation?